### PR TITLE
Remove forward references to SCMP

### LINKED ISSUE #8: SCMP - Be careful about forward references
>  SCION also includes its own protocol to communicate failures to endpoints,
   the SCION Control Message Protocol (SCMP).  This protocol will be
   described in a separate document, which will follow later.

Text like this doesn't stand on its own.

### PR DESCRIPTION
**Pull request by jiceatscion**
Fixes #8

### CODE REVIEW

File: draft-dekater-scion-dataplane.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ SCION leverages source-based path selection, where path information is embedded
 
 **Path Transparency**: Path transparency is a property of a network architecture that gives endpoints full visibility over the network paths their packets are taking. Path transparency is weaker than path control.
 
-**SCMP**: SCION Control Message Protocol. SCMP is used for signaling connectivity problems, analogous to the Internet Control Message Protocol (ICMP). SCMP provides network diagnostic and error messages.
+**SCMP**: Future SCION Control Message Protocol. SCMP is intended to be analogous to the Internet Control Message Protocol (ICMP).
```
Comment (nicorusti) [inline draft-dekater-scion-dataplane.md:207]:
```suggestion
```

File: draft-dekater-scion-dataplane.md [hunk 2]
```diff
@@ -1445,24 +1442,25 @@ SCION attempts to take the IANA's assigned Internet protocol numbers into consid
 
 The protocol numbers are used in the SCION header to identify the next level protocol.
 
+SCION reserves a number for a future protocol: the SCION Control Message Protocol (SCMP). Support for this protocol is optional and it has not yet been formally specified. A work-in-progress specification exists: [the scion.org SCMP specification document](https://docs.scion.org/en/latest/protocols/scmp.html).
```
Comment (nicorusti) [inline draft-dekater-scion-dataplane.md:1445]: Maybe we could remove this comment and leave it in the table? I think we have the same problem for DrKey somewhere else

**Review by nicorusti — CHANGES_REQUESTED** [review]

File: draft-dekater-scion-dataplane.md (hunk 1 above)
Comment (jiceatscion) [reply draft-dekater-scion-dataplane.md:207]: I can't figure what you're suggesting. The review UI seems to say that you want to remove the new text entirely as well as the old text. Is that what you're suggesting?

File: draft-dekater-scion-dataplane.md (hunk 2 above)
Comment (jiceatscion) [reply draft-dekater-scion-dataplane.md:1445]: I was acting on Eliot's suggestion: mention the protocol informally. If we just leave in the table with no explanation, we *will* get the question "what is this?". What is the motivation for your suggestion?

**Review by nicorusti — APPROVED** [review]
Looks good, I'll ask Eliot if the reference works, If I don't hear back in a few days I'd berge and just leave the corresponding issue open

**Comment by nicorusti** [comment]
Eliot gave us a thumbs up for the current reference, merged.

**Pull request merged**